data/repositories.toml

```diff
@@ -280,6 +279,7 @@ repositories = [
   'github.com/hubtype/botonic',
   'github.com/hudson-and-thames/mlfinlab',
   'github.com/Hypfer/Valetudo',
+  'https://github.com/Heptagram-Bot/Heptagram',
```

Comment:
Hi @j-dogcoder 
You need to fix three things:
- Please don't remove link of other repositories
- You don't need to add `https://` in a repository link
- It's still not in the lexicograpthic order 

What I'd suggest you is to do things correctly and quickly is:
1. Delete your fork and fork the repo again
2. Create new branch
3. List your repo in line number 276 of data/repositories.toml
4. Submit the PR

Reply:
Ok! Will do!